infra/build/developer-tools-krm/Dockerfile

```diff
@@ -64,3 +64,7 @@ RUN rm -rf /build
 # Add dind helper for prow
 ADD ./build/scripts/runner.sh /usr/local/bin/
 ADD ./build/scripts/prow_entrypoint.sh /usr/local/bin/
+
+# cookiecutter is used for terraform-google-module-template tests
+# https://github.com/terraform-google-modules/terraform-google-module-template
+RUN pip install cookiecutter
```

Comment:
Do we need this since this is only used in KRM tests?

Reply:
For now I moved to just the krm image as that image is used for github.com/terraform-google-modules/terraform-google-module-template CI

A future option would be to do something specific in the repo so this could be removed entirely.